vizro-core/docs/pages/user_guides/custom_actions.md

```diff
@@ -66,7 +66,7 @@ The following example shows how to create a custom action that postpones executi
 ### Interacting with dashboard inputs and outputs
 When a custom action needs to interact with the dashboard, it is possible to define `inputs` and `outputs` for the custom action.
```

Comment:
I think we have a perfect match when users search for the keyword `callbacks`. There is only one matching result that leads to the beginning of `/user_guides/actions` documentation. 

<img width="757" alt="image" src="https://github.com/mckinsey/vizro/assets/108530920/e0e83123-d4de-4569-9115-e6081d356ea4">

Reply:
Thanks for checking! Somehow I get different results than you, for me the first result are custom components and then these action docs. This means if someone would look for callbacks or how to write their own callbacks, they currently won't find the right docs by keyword search. I think it would be good to mention it in the docs for the custom actions, such that it pops up as one of the first results when looking for "writing own callback", "callbacks" etc.

Reply:
We got different results because I searched for `"calbacks"` and I assume you searched for `"callback"`. 😄 
Even we add a `"callback"` key word in the custom action `.md` file, the "How to create custom component" result will appear as the first suggested answer.

Reply:
It's added to the `custom_actions.md` the `"callbacks"` search now looks like this:
<img width="757" alt="image" src="https://github.com/mckinsey/vizro/assets/108530920/abca4362-e115-4b7d-9760-a712651ad258">